search/query/date_range.go

```diff
@@ -135,16 +191,33 @@ func (q *DateRangeQuery) Field() string {
 }
 
 func (q *DateRangeQuery) Searcher(ctx context.Context, i index.IndexReader, m mapping.IndexMapping, options search.SearcherOptions) (search.Searcher, error) {
-	min, max, err := q.parseEndpoints()
-	if err != nil {
-		return nil, err
-	}
-
 	field := q.FieldVal
 	if q.FieldVal == "" {
 		field = m.DefaultSearchField()
 	}
-
+	if q.rawStart != "" || q.rawEnd != "" {
```

Comment:
Hmm, should probably do this string job only when index mapping for this field has store enabled and the query requested for this field ?

Reply:
the object here is a query object right? like this check is just done to check if the unmarshaller used was my custom one (which is the only place where raw start and raw end is actually set) or the default one used in User Path. these 2 params are not exported so bleve only users cant see this